storage/storage_image.go

```diff
@@ -137,10 +137,9 @@ func (s *storageImageDestination) putBlob(stream io.Reader, blobinfo types.BlobI
 		Digest: "",
 		Size:   -1,
 	}
-	// Try to read an initial snippet of the blob.
```

Comment:
I wouldn’t mind preserving this comment, especially when the comment below refers to “the initial snippet” from this comment. (Absolutely non-blocking.)